Makefile

```diff
@@ -2,6 +2,9 @@
 # Configuration
 #############################################################
 
+# Allow users to select a different cross compiler.
+CROSS_COMPILE ?= riscv64-unknown-elf-
```

Comment:
It's probably more ergonomic to not expect the user to include the trailing '-'

Reply:
Linux makes you stick the "-" in there, which is where I stole the variable from.  I'm OK changing it if you write the code :)

Reply:
If Linux expects it I'm fine with the principle of least surprise.

Reply:
OK, I just did it :)